torch/_ops.py

```diff
@@ -309,7 +310,27 @@ def dispatch(self, dispatch_key, *args, **kwargs):
                 return handler(mode, *args, **kwargs)
 
         functionality_key = torch._C._to_functionality_key(dispatch_key)  # type: ignore[attr-defined]
-        if functionality_key in mode_stack_per_key():
+        if functionality_key == torch._C.DispatchKey.PreDispatch:
```

Comment:
The code here is for the HigherOrderOp case - are there any tests for higher order ops + pre-dispatch functionalization?

Reply:
There are some old tests in test_proxy_tensor.py. I think it is ok because it means even tho predispatch functionalization for higher order op doesn't work, it at least doesn't break the old tests. So no regressions.